test/e2e/support/setup-test-framework.js

```diff
@@ -116,6 +116,12 @@ function observeConsoleLogging() {
 			return;
 		}
 
+		// Viewing posts on the front end can result in this error, which
+		// has nothing to do with Gutenberg.
+		if ( text.includes( 'net::ERR_UNKNOWN_URL_SCHEME' ) ) {
```

Comment:
This seems to be a jQuery thing that happens on the front end, and we sometimes would get hit by it, but sometimes the test assertions would complete quickly enough for it not to make the test fail.

Reply:
Is there an issue anywhere that tracks that? I don't quite follow.

Reply:
I'm not sure what causes it yet, it seems to be unrelated to Gutenberg, and happens when you view the published post on the front end. I'll investigate further after this merge.